src/resources.js

```diff
@@ -37,4 +38,17 @@ function _executable(name) {
 }
 const executable = memoize(_executable);
 
-module.exports = { get, executable };
+function _wslify(path) {
```

Comment:
I would be much more comfortable doing something like this natively via WSL, by setting the path we want in an environment variable, setting `WSLENV` for that to `/up` (to translate paths), and then just printing the result from Linux.  It's a bit slower, but it ensures we use the correct code.

Reply:
Done, and found some flaws in the code: 
1. Don't check to see if the file exists, because we might be upgrading it.
2. Don't move the file, copy it (it might be on a readonly partition, but there's no reason to move it anyway)
3. I wasn't awaiting on the moves, causing a race condition